File: BackUp/2013/version-2013-07-03/AttackoftheRGB/Assets/C#/redEnemySpawn.cs
using UnityEngine;
using System.Collections;

public class redEnemySpawn : MonoBehaviour 
{
	public Transform enemy;
	public int count = 0;
	public double savedTime = 0;
	public bool isActive = false;

	int randomNumber;

	private void Start()
	{
		InvokeRepeating ("PickRandomNumber", 1.0f,1.0f);
	}

	private void PickRandomNumber()
	{
		randomNumber = Random.Range(0,10);
	}

	private void Update()
	{
		//print(randomNumber);
		if (randomNumber == 3)
		{
			isActive = true;
		
		}
		if(isActive)
		{
			//print(count);
			int seconds = (int)Time.time;
			double time = (seconds % 2);
			
			if(isActive)
			{
				if(time == time*200 && count < 2)
				{
					Spawn(seconds);
					
				}
			}
			isActive = false;
			
		}
		else
		{
			isActive = false;
		}
	}

	private void Spawn(int seconds)
	{
		if(seconds != savedTime)
		{
			Transform ENEMY = (Transform)Instantiate(enemy, transform.position, Quaternion.identity);
			ENEMY.gameObject.tag = "Enemy";
			count++;
			savedTime = seconds;		
		}
	}
}

File: BackUp/2013/version-2013-07-03/AttackoftheRGB/Assets/Scripts/JavaScript/redEnemySpawn.js
#pragma strict

var enemy : Transform;

//var red : Transform;
//var green : Transform;
//var blue : Transform;

var count  = 0;
var savedTime = 0;
var isActive = false;

var randomNumber;

function Start () 
{
	InvokeRepeating ("PickRandomNumber", 1.0, 1.0);
	
}

function PickRandomNumber()
{
	//yield WaitForSeconds(2);
	randomNumber = Random.Range(0,10);
}

function Update () 
{
	//print(randomNumber);
	if (randomNumber == 3)
    {
    	isActive = true;
    
	}
	if(isActive)
	{
		//print(count);
		var seconds : int = Time.time;
		var time = (seconds % 2);
		
		if(isActive)
		{
			if(time == time*200 && count < 2)
			{
				Spawn(seconds);
				
			}
		}
		isActive = false;
		
	}
	else
	{
		isActive = false;
	}
}

function Spawn(seconds)
{
	if(seconds != savedTime)
	{
		var ENEMY = Instantiate(enemy, transform.position, Quaternion.identity);
		ENEMY.gameObject.tag = "Enemy";
		count++;
		savedTime = seconds;
		
	}
}